ocaml/parsing/jane_syntax.ml

```diff
@@ -339,16 +364,17 @@ module Strengthen = struct
   type nonrec module_type =
     { mty : Parsetree.module_type; mod_id : Longident.t Location.loc }
 
-  let extension_string = Language_extension.to_string Module_strengthening
+  let { erasability; feature } =
+    make_for_language_extension Module_strengthening
 
   (* Encoding: [S with M] becomes [functor (_ : S) -> (module M)], where
      the [(module M)] is a [Pmty_alias].  This isn't syntax we can write, but
      [(module M)] can be the inferred type for [M], so this should be fine. *)
 
   let mty_of ~loc { mty; mod_id } =
     (* See Note [Wrapping with make_entire_jane_syntax] *)
-    AST.make_entire_jane_syntax Module_type ~loc extension_string (fun () ->
-      Ast_helper.Mty.functor_ (Named (Location.mknoloc None, mty))
+    AST.make_entire_jane_syntax Module_type ~loc erasability feature (fun () ->
+        Ast_helper.Mty.functor_ (Named (Location.mknoloc None, mty))
```

Comment:
```suggestion
      Ast_helper.Mty.functor_ (Named (Location.mknoloc None, mty))
```
this is a function applied to the next line, so it needs to have intermediate indentation